CONTRIBUTING.md

```diff
@@ -0,0 +1,200 @@
+# Contributing to Zen MCP Server
+
+Thank you for your interest in contributing! This guide explains how to set up the development environment and contribute to the project.
+
+## Development Setup
+
+1. **Clone the repository**
+   ```bash
+   git clone https://github.com/BeehiveInnovations/zen-mcp-server.git
+   cd zen-mcp-server
+   ```
+
+2. **Create virtual environment**
+   ```bash
+   python -m venv venv
+   source venv/bin/activate  # On Windows: venv\Scripts\activate
+   ```
+
+3. **Install dependencies**
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+## Testing Strategy
+
+### Two Types of Tests
+
+#### 1. Unit Tests (Mandatory - No API Key Required)
+- **Location**: `tests/test_*.py` (except `test_live_integration.py`)
+- **Purpose**: Test logic, mocking, and functionality without API calls
+- **Run with**: `python -m pytest tests/ --ignore=tests/test_live_integration.py -v`
+- **GitHub Actions**: ✅ Always runs
+- **Coverage**: Measures code coverage
+
+#### 2. Live Integration Tests (Optional - API Key Required)
+- **Location**: `tests/test_live_integration.py` 
+- **Purpose**: Verify actual API integration works
+- **Run with**: `python tests/test_live_integration.py` (requires `GEMINI_API_KEY`)
+- **GitHub Actions**: 🔒 Only runs if `GEMINI_API_KEY` secret is set
+
+### Running Tests
+
+```bash
+# Run all unit tests (CI-friendly, no API key needed)
+python -m pytest tests/ --ignore=tests/test_live_integration.py -v
+
+# Run with coverage
+python -m pytest tests/ --ignore=tests/test_live_integration.py --cov=. --cov-report=html
```

Comment:
Please review against current README, a lot of this was removed since it's no longer relevant, I slimmed the readme down quite a lot